relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -331,6 +334,17 @@ pub(crate) fn extract_tags(
             span_tags.insert(SpanTagKey::Group, span_group);
 
             let truncated = truncate_string(scrubbed_desc, config.max_tag_value_size);
+            if span_op.starts_with("resource.") {
+                if let Some(ext) = truncated
+                    .rsplit('/')
+                    .next()
+                    .and_then(|last_segment| last_segment.rsplit_once('.'))
```

Comment:
```suggestion
                    .unwrap_or(truncated)
                    .rsplit_once('.')
```

Then this works also on resources which do not contain a slash